spec/models/collection_item_spec.rb

```diff
@@ -110,23 +112,37 @@
         it "enqueues the collection for reindex" do
           expect do
             item.update!(collection_approval_status: :rejected)
-          end.to add_to_reindex_queue(collection, :background)
+          end.to add_to_reindex_queue(collection, :background) &
+                 add_to_reindex_queue(parent_collection, :background)
+        end
+      end
+
+      context "when collection item is accepted" do
```

Comment:
I think we're missing tests for changing the `user_approval_status`. We could maybe also use one for setting `collection_approval_status` to `:unreviewed`, but that's probably not as important as a test that confirms unreviewed items aren't included in the counts.